sdk/maps/azure-maps-search/azure/maps/search/_generated/models/__init__.py

```diff
@@ -6,228 +6,116 @@
-    from ._models import GeoJsonFeatureCollectionData  # type: ignore
-    from ._models import GeoJsonFeatureData  # type: ignore
-    from ._models import GeoJsonGeometry  # type: ignore
-    from ._models import GeoJsonGeometryCollection  # type: ignore
-    from ._models import GeoJsonGeometryCollectionData  # type: ignore
-    from ._models import GeoJsonLineString  # type: ignore
-    from ._models import GeoJsonLineStringData  # type: ignore
-    from ._models import GeoJsonMultiLineString  # type: ignore
-    from ._models import GeoJsonMultiLineStringData  # type: ignore
-    from ._models import GeoJsonMultiPoint  # type: ignore
-    from ._models import GeoJsonMultiPointData  # type: ignore
-    from ._models import GeoJsonMultiPolygon  # type: ignore
-    from ._models import GeoJsonMultiPolygonData  # type: ignore
-    from ._models import GeoJsonObject  # type: ignore
-    from ._models import GeoJsonPoint  # type: ignore
-    from ._models import GeoJsonPointData  # type: ignore
-    from ._models import GeoJsonPolygon  # type: ignore
-    from ._models import GeoJsonPolygonData  # type: ignore
-    from ._models import GeometryIdentifier  # type: ignore
-    from ._models import LatLongPairAbbreviated  # type: ignore
-    from ._models import OperatingHours  # type: ignore
-    from ._models import OperatingHoursTime  # type: ignore
-    from ._models import OperatingHoursTimeRange  # type: ignore
-    from ._models import PointOfInterest  # type: ignore
-    from ._models import PointOfInterestCategory  # type: ignore
-    from ._models import PointOfInterestCategorySet  # type: ignore
-    from ._models import PointOfInterestCategoryTreeResult  # type: ignore
-    from ._models import Polygon  # type: ignore
-    from ._models import PolygonResult  # type: ignore
-    from ._models import ReverseSearchAddressBatchItem  # type: ignore
-    from ._models import ReverseSearchAddressBatchItemResponse  # type: ignore
-    from ._models import ReverseSearchAddressBatchProcessResult  # type: ignore
-    from ._models import ReverseSearchAddressResult  # type: ignore
-    from ._models import ReverseSearchAddressResultItem  # type: ignore
-    from ._models import ReverseSearchCrossStreetAddressResult  # type: ignore
-    from ._models import ReverseSearchCrossStreetAddressResultItem  # type: ignore
-    from ._models import SearchAddressBatchItem  # type: ignore
-    from ._models import SearchAddressBatchItemResponse  # type: ignore
-    from ._models import SearchAddressBatchResult  # type: ignore
-    from ._models import SearchAddressResult  # type: ignore
-    from ._models import SearchAddressResultItem  # type: ignore
-    from ._models import SearchAlongRouteRequest  # type: ignore
-    from ._models import SearchInsideGeometryRequest  # type: ignore
-    from ._models import SearchSummary  # type: ignore
-    from ._models import Viewport  # type: ignore
+from ._models import Address
```

Comment:
All these models need to be added to the public API.

Reply:
You mean to expose models outside of _generated dir? In the azure.maps.search.models package?

Reply:
I would no longer use the `_generated` directory at all, instead we should just use a fully generated SDK. Then all the necessary models will automatically be public.

Reply:
You mean, just to move everything from generated directory into azure.maps.search instead? And to add **custom** models in the same models.py file (where are now generated models)?

Reply:
But just to clarify, we do need custom models to wrap up everything the same way as for V1, right?
And for package structure:
azure.maps.search:
- _commons
- aio
- operations
- models
- client.py - from generated dir
- search_client.py - custom client used as a wrapper
... etc.

Is this the package structure you would suggest?

Reply:
Hi @annatisch , like Lazar's question, what does it mean to use a fully generated SDK and do we have other services for us to reference?